app/src/pages/settings/AnnotationConfigDialog.tsx

```diff
@@ -0,0 +1,424 @@
+                      >
+                        <Controller
+                          control={control}
+                          name={`values.${index}.label`}
+                          rules={{
+                            required: "Category label is required",
+                          }}
+                          render={({ field, fieldState: { error } }) => (
+                            <TextField
+                              {...field}
+                              aria-label={`Value ${index + 1}`}
+                              isInvalid={!!error}
+                            >
+                              <Input placeholder="Category label" />
+                              <FieldError>{error?.message}</FieldError>
+                            </TextField>
+                          )}
+                        />
+                        <Controller
+                          control={control}
+                          name={`values.${index}.score`}
+                          rules={{
+                            required: "Score is required",
+                          }}
+                          render={({ field, fieldState: { error } }) => (
+                            <NumberField
+                              {...field}
+                              value={
+                                typeof field.value === "number"
+                                  ? field.value
+                                  : undefined
+                              }
+                              aria-label={`Score ${index + 1}`}
+                              isInvalid={!!error}
+                            >
+                              <Flex
+                                direction="row"
+                                gap="size-100"
+                                alignItems="center"
+                              >
+                                <Input placeholder={`${index}`} />
+                              </Flex>
+                              <FieldError>{error?.message}</FieldError>
+                            </NumberField>
+                          )}
+                        />
+                        <Button
+                          type="button"
+                          onPress={() => remove(index)}
+                          variant="quiet"
+                        >
+                          <Icon svg={<Icons.TrashOutline />} />
+                        </Button>
+                      </Flex>
+                    ))}
+                    <Button
+                      type="button"
+                      onPress={() => {
+                        append({ label: "", score: fields.length });
+                      }}
+                    >
+                      Add category
+                    </Button>
+                  </>
+                )}
+              </Flex>
+            </View>
+            <View
+              paddingX="size-200"
+              paddingY="size-100"
```

Comment:
<img width="968" alt="Screenshot 2025-03-24 at 9 20 41 AM" src="https://github.com/user-attachments/assets/957d8707-b454-4c4b-8b50-ae4975ad49c6" />
When using a full-sized button I sorta feel like the padding should be symmetrical on the X and Y. I know that's inconsistent with some of the other places. We probably just need to figure out the standard.